### DISCUSSION TITLE
Hello World! ;)

### DISCUSSION THREAD

**Discussion by MoAmrYehia** [OpenMined/courses#14] [Introduce Yourself]
Hi, I am Mohamed Amr a ML Engineer from Egypt. Feel free to contact me on [LinkedIn](https://www.linkedin.com/in/mamryehia/).

**Comment by yemikifouly** [comment]
Hi Mohamed! 🙂 Welcome to the discussion board! Have you learned about Private AI before or will this be your introduction to it?

**Reply by MoAmrYehia** [reply]
Hello Yemissi, Thanks for your reply. Yeah, I am familiar with the topic since I was selected 2 years ago to enroll Secure and Private AI Scholarship Challenge from Facebook.

**Reply by yemikifouly** [reply]
How interesting Mohamed! I was in that scholarship too! 🙂 Small world! 😀 How is the course going?

**Reply by MoAmrYehia** [reply]
It is amazing! I started today as I have been working on some tasks, however, I am about to finish lesson two.

**Reply by yemikifouly** [reply]
How great!!! 🙂 I hope you are liking the content so far.
